main.py

```diff
@@ -441,6 +441,7 @@ def delete_circuit(self, request: Request) -> JSONResponse:
             evc.archive()
             evc.remove_uni_tags()
             evc.sync()
+        self.circuits.pop(circuit_id)
```

Comment:
Notice that multiple threads might still execute this pop here. Under normal circumstances it's rare that too many concurrent deletions will be sent for the same EVC, but a few can enter on line 428 and then keep waiting for `evc.lock`, and finally they execute this line here 444 resulting in a `KeyError` exc. 

```
  File "/home/viniarck/repos/napps/napps/kytos/mef_eline/main.py", line 445, in delete_circuit
    self.circuits.pop(circuit_id)
KeyError: '2badb4f116e043'
```

You'll need to `.pop(circuit_id, None)` or catch `KeyError`, and then we'll need to be careful to only emit the event `"deleted"` once, so you need to know if indeed it the evc was popped, otherwise this will result in redundant events for other NApps such as `telemetry_int`. 

The other `evc.*` removals methods if they end up executing more than once as far as I followed, shouldn't cause other issues, so that's OK.

Reply:
No longer using lock. Deleted the loading of EVCs from consistency check instead.